components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -172,15 +181,17 @@
 					}
 				}
 			}
-
 			Thread.sleep(500);
 			return map;
-		} catch (Exception e) {
+		} catch (Throwable e) {
+			//catch throwable since there could be execution plan validation exceptions
 			errMsg = "Error occurred while processing. " + e.getMessage();
 			log.error(errMsg, e);
 			throw new Exception(errMsg, e);
 		} finally {
-			executionPlanRuntime.shutdown();
+			if (!executionPlanRuntime.equals(null)) {
```

Comment:
Yes sure, thank you for pointing out. Fixed in PR:https://github.com/wso2/carbon-event-processing/pull/179